docs/visualization/helper_api/lttng_event_filter.md

```diff
@@ -43,5 +43,4 @@ lttng = Lttng('/path/to/ctf', event_filters=[
 ]) # Filtering from 5 seconds to 10 seconds after the start of the measurement
 ```
 
-Multiple event_filters can be specified.  
-If there are multiple event_filters have been given, all filters will be applied to the selected interval only.
+Multiple event_filters can be specified.
```

Comment:
@takeshi-iwanari 
I adapted your comment.
https://github.com/tier4/CARET_doc/pull/95#discussion_r999237207
> all filters will be filtered -> all filters will be applied sounds more natural.
> The meaning of the specified interval is not clear

I deleted the sentence.